projects/common/src/utilities/url/url-utilities.ts

```diff
@@ -0,0 +1,29 @@
+import { Dictionary, isEmpty } from 'lodash';
+
+export const getQueryParamStringFromObject = (params: Dictionary<string | number>): string => {
+  try {
+    const paramString: string = Object.entries(params)
+      .map(([key, value]) => `${key}=${value}`)
+      .filter(item => item)
+      .join('&');
+
+    return isEmpty(paramString) ? '' : paramString;
+  } catch (error) {
+    return ``;
+  }
+};
+
+export const getQueryParamObjectFromString = (query: string): Dictionary<string> => {
+  try {
```

Comment:
we don't have a usecase for this method today. you can remove this.

Reply:
removed